src/qibolab/dummy.py

```diff
@@ -219,10 +304,16 @@ def create_dummy():
     channels |= (Channel(f"drive-{i}", port=instrument[f"drive-{i}"]) for i in range(nqubits))
     channels |= (Channel(f"flux-{i}", port=instrument[f"flux-{i}"]) for i in range(nqubits))
     channels |= Channel("twpa", port=None)
+    # FIXME: Issues with the names if they are strings maybe
+    channels |= (
+        Channel(f"flux_coupler-{c}", port=instrument[f"flux_coupler-{c}"])
+        for c in itertools.chain(range(0, 2), range(3, 5))
```

Comment:
I usually like this a lot, but in this case maybe the explicit spell out would be simpler, since it would be just:
```suggestion
        (0, 1, 3, 4)
```

Reply:
I know you do, it come from your Zurich suggestions hahah